.github/workflows/CI.yml

```diff
@@ -0,0 +1,520 @@
+name: CI/CD
+
+on: [push, workflow_dispatch]
+
+jobs:
+  ci_ubuntu:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ['ubuntu-18.04', 'ubuntu-20.04']
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '2.7.18'
+
+      - name: Install libraries on ubuntu 18
+        if: ${{ (matrix.os == 'ubuntu-18.04') }}
+        run: sudo apt-get install netcdf-bin libnetcdf-dev python-dev
```

Comment:
I suspect this line needs to also contain `libgtk2.0-0` as that seems to be a missing dependency for Ubuntu 18

Reply:
Since it compiles correctly, I would think that `libgtk2.0-0` is already installed on the ubuntu 18 GitHub runner. Do you think that installing it there would make the created DEB file not to depend on `libgtk2.0-0`?

Otherwise, I have added the dependency to the `control` file, so if MDANSE is installed with apt, `libgtk2.0-0` should get installed automatically if it isn't already.

Reply:
yeah, adding it to `control` is what I should have said.